### PR TITLE
[github] Add PR template & update issue templates

### PR DESCRIPTION
**Pull request by TheCatLady**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<!-- Please describe the changes included in this PR here. -->
+
+# To-Dos
+
+<!-- Before submitting this PR, please lint and build your changes locally. -->
+<!-- Add an 'x' between the brackets to mark each checkbox as checked. -->
+
+- [ ] Lint and test with `tox`
+
```
Comment (aquamatthias) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
- [ ]  PR includes a test that will fail if the code is reverted, but the test is not.

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Issues Fixed
+
+<!-- If this PR will fix/resolve an open issue on the repository, please reference it below. -->
+<!-- Otherwise, please delete this section. -->
+
+- Fixes #XXXX
```
Comment (aquamatthias) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Please move the example into the html comment.
So nothing is rendered if lazy people forget to delete this part.

**Review by aquamatthias — COMMENTED** [review]
2 small additions - otherwise looks fine to me.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (TheCatLady) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: The heading would need to be moved as well, in that case. I don't think there's a good way to handle this until GitHub supports pull request forms.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (TheCatLady) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: How about:

```
- [ ] Test coverage for new code
```

Your suggestion is a little difficult to understand.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (TheCatLady) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: There is also no harm in the user not deleting this section. But there is a lot of benefit to making it easy for the user to link related issues. :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (aquamatthias) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: It is probably related but different.
If it is too complex to understand - let's not add it.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (aquamatthias) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: I prefer to hint at what a good PR description could look like - I assume #XXXX will get fixed often.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Description
+
+<!-- Please describe the changes included in this PR here. -->
+
+# To-Dos
+
+<!-- Before submitting this PR, please lint and build your changes locally. -->
+<!-- Add an 'x' between the brackets to mark each checkbox as checked. -->
+
+- [ ] Test coverage for new code
```
Comment (aquamatthias) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Somewhat unclear to me. Maybe something like:

- [ ] Added or changed functionality is covered by tests

wdyt?

**Review by aquamatthias — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (TheCatLady) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: How about:

```
- [ ] Test coverage added for new or updated functionality
```

Comment (aquamatthias) [reply .github/PULL_REQUEST_TEMPLATE.md]: Like it

**Review by aquamatthias — APPROVED** [review]

**Pull request merged**